feldman_lab_to_nwb/convert_feldman/feldman_electrode_widget.py

```diff
@@ -1,51 +1,65 @@
 import numpy as np
 
-from ipywidgets import widgets
+from ipywidgets import widgets, Layout
 from pynwb.misc import Units
 from nwbwidgets import default_neurodata_vis_spec
 from nwbwidgets.misc import PSTHWidget
 from nwbwidgets import base
 import plotly.graph_objects as go
 
+from .feldman_electrode_widget_utils import calculate_response
 
-BACKGROUND_COLOR = "#9FE19D"
-ELECTRODE_COLOR = "#d81700"
-DETECTED_COLOR = "#002bd8"
-SELECTED_COLOR = "#fccb00"
+ELECTRODE_COLOR = "#000000"
+
+ELECTRODE_SIZE = 3
+DETECTED_SIZE = 10
+SELECTED_SIZE = 20
 
 
 class ElectrodePositionSelector(widgets.VBox):
 
     def update_point(self, trace, points, selector):
         pass
 
-    def __init__(self, electrodes):
+    def __init__(self, electrodes, psth_info):
```

Comment:
It's not clear what "psth_info" should be. Can you please add a docstring?

Reply:
Renamed more appropriately to `responsitivity_params`; the PSTH happens to be where the info is coming right now, but in its current form it's already generalized beyond that.

Added docstring for its form. It also doesn't have to use the dictionary packing if that's adding confusion, I was just doing that for convenience in passing between the widgets.

Reply:
Dictionary compression is no longer used here, just direct keyword arguments.